### PR TITLE
Preserve Data

### PR DESCRIPTION
**Pull request by kensteph**
Hi there 👋!
In this PR I : 
✅ Create an Author Module.
✅ Preserve author data in JSON file.
✅ Create a Game Module.
✅ Implement list_all_games method.
✅ Implement add_game method.
✅ Implement preseve_game method.
✅ Implement load_games and load_authors.
✅ Implement list_all_authors.

### CODE REVIEW

File: modules/game.rb [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    # AUTHOR
+    puts "\n"
+    puts 'Please provide these information about the author:'
+    print 'First name : '
+    first_name = gets.chomp
+    print 'Last name : '
+    last_name = gets.chomp
+    new_author = Author.new(first_name, last_name)
+    @authors << new_author
+    preseve_game
```
Comment (dfsalam) [inline modules/game.rb]: I kindly recommend you fix the typo on preserve to make the code more readable.

**Review by dfsalam — CHANGES_REQUESTED** [review]
Hi @kensteph 
A great approach to solve the problem. To improve the readability of your code I recommend you correct the typo.

File: modules/game.rb (hunk 1 above)
Comment (kensteph) [reply modules/game.rb]: Thank you @dfsalam for your review.

**Review by dfsalam — APPROVED** [review]
# STATUS: APPROVED 💯

**Pull request merged**